lib/ex_aws/ses.ex

```diff
@@ -372,4 +373,7 @@ defmodule ExAws.SES do
   defp do_flatten_attrs({val, path}) do
     {camelize_key(path), val}
   end
+
+  defp put_not_nil(map, _, nil), do: map
+  defp put_not_nil(map, key, value), do: map |> Map.put(key, value)
```

Comment:
We could be a little bit more explicit by naming it `put_if_not_nil/3`. What do you think?